lib/modules/versioning/npm/range.ts

```diff
@@ -64,7 +64,10 @@ export function getNewValue({
   currentVersion,
   newVersion,
 }: NewValueConfig): string | null {
-  if (rangeStrategy === 'bump' && isSemVerXRange(currentValue)) {
+  if (
+    !['pin', 'update-lockfile'].includes(rangeStrategy) &&
```

Comment:
bump wasnt supported, and it will still return null.
the test for bump wasnt changed.

bump will be evaluated the same, but this time replace and widen will be returning null as well